examples/bert/README.md

```diff
@@ -212,3 +215,28 @@ In this simple example, the hyperparameters to be tuned are provided as a `dict`
 library). The example runs for 3 trials to find the best hyperparam settings. The final model is
 saved in `output_dir` provided by the user. More  information about the libary can be 
 found at [Hyperopt](https://github.com/hyperopt/hyperopt).
+
+### Hyperparamter tuning with Neural Network Intelligence (NNI)
+
+To run this example, please install `nni` by issuing the following command
+
+```commandline
+python3 -m pip install --upgrade nni
```

Comment:
Since texar-pytorch supports python 3+ only, so we can simply specify `python -m` here.